src/clj/rems/api/user_settings.clj

```diff
@@ -3,7 +3,19 @@
             [rems.api.schema :as schema]
             [rems.db.user-settings :as user-settings]
             [rems.util :refer [getx-user-id get-user-id]]
-            [ring.util.http-response :refer :all]))
+            [ring.util.http-response :refer :all]
+            [schema.core :as s])
+  (:import (org.joda.time DateTime)))
+
+(s/defschema GetUserSettings
+  {:language s/Keyword
+   :notification-email (s/maybe s/Str)
+   (s/optional-key :ega) {:api-key-expiration-date DateTime}})
```

Comment:
I know we want to be able to deduplicate the db and api models so this is nice, but since the models are the same at the moment, could this just be a `(def GetUserSettings user-settings/UserSettings)` for now?

I bet we went over this previously but I've forgotten our conversation.